app/controllers/defence_requests_controller.rb

```diff
@@ -10,7 +10,7 @@ class DefenceRequestsController < BaseController
   helper_method :defence_request_path_with_tab
 
   def show
-    @tab = params[:tab]
+    @tab = params[:tab] && %(interview case-details).include?(params[:tab]) ?  params[:tab] : nil
```

Comment:
Also whenever params[:tab] is checked the value must be whitelisted - not like https://github.com/ministryofjustice/defence-request-service/pull/214/files#diff-5e2ba0f9f4b5b18a5e05aebd97940c33R45 where the value is directly passed to a route helper (security risk)